artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/ActiveMQChannelHandler.java

```diff
@@ -89,6 +90,11 @@ public void exceptionCaught(final ChannelHandlerContext ctx, final Throwable cau
       if (!active) {
          return;
       }
+
+      if (cause instanceof ReadTimeoutException) {
+         ActiveMQClientLogger.LOGGER.error("Connection without communication timeout has expired.");
```

Comment:
Should probably have a specific error code defined in ActiveMQClientLogger and use that.